agent/util-scripts/test-bin/rpmdev-vercmp

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+args="${@}"
+echo "${0} ${args}" >> ${_testlog}
```

Comment:
I understand what is going on in this script because I just finished reading `agent/util-scripts/test-bin/rpm`, but, if someone were to come here fresh, I imagine they would appreciate some explanation.